src/Components/Cluster/Cluster.js

```diff
@@ -29,16 +32,29 @@ export const Cluster = ({ cluster, match }) => {
           <Loading />
         </Main>
       )}
-      {isError && (
-        <Main>
-          <MessageState
-            title={intl.formatMessage(messages.noRecsError)}
-            text={intl.formatMessage(messages.noRecsErrorDesc)}
-            icon={ExclamationCircleIcon}
-            iconStyle={{ color: globalDangerColor100.value }}
-          />
-        </Main>
-      )}
+      {isError &&
+        (cluster.error?.status === 400 ? (
+          <Main>
+            <NoRecsState
+              title={intl.formatMessage(messages.noRecsFoundError)}
+              text={intl.formatMessage(messages.noRecsFoundErrorDesc)}
+              icon={InfoCircleIcon}
+              iconStyle={{ color: globalInfoColor100.value }}
+              variant="large"
+              link="https://docs.openshift.com/container-platform/latest/support/getting-support.html"
+              linktext=" OpenShift documentation"
+            />
+          </Main>
+        ) : (
+          <Main>
+            <MessageState
```

Comment:
Could, for the "404 case", `MessageState` can be utilized too? Did you try passing `text` param like:
```
...
text={<React.Fragment>{intl.formatMessage(messages.noRecsErrorDec}<a href="https://docs.openshift.com/container-platform/latest/support/getting-support.html">OpenShift documentation</a></React.Fragment>}
...

```

Reply:
It worked! Haha! I will remove unnecessary component I created before